wordpress-mobile/WordPress-iOS
# Rich Notifications: Enable rich notifications for the Jetpack app

Part of #19361 

## Description
Creates `JetpackNotificationServiceExtension` which is a replica of the `WordPressNotificationServiceExtension`.

## Testing Instructions

1. Install Jetpack on a device.
2. Enable notifications
3. Sign in to your account
4. From a different account, comment on one of your posts
5. You should receive a notification
6. Make sure the notification includes the comment's content as well as the commenter and the post title.

_P.S: Testing only one type of notification is sufficient since the JP extension runs the exact same code as the WP one._

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

6. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
